client/js/nonprofits/donate/info-step.js

```diff
@@ -38,8 +37,14 @@ function init(donation$, parentState) {
   state.showDedicationForm$ = flyd.map(()=> false, state.submitDedication$)
 
   // Save donor supporter record
-  state.supporterFields = supporterFields.init({required: {email: true}}, parentState.params$)
-  state.savedSupp$ = flyd.flatMap(postSupporter , flyd.map(formatFormData, state.submitSupporter$))
+  let formData$ = flyd.map(formatFormData, state.submitSupporter$)
+
+  state.supporterFields = supporterFields.init({required: {email: true}}, state.params$)
+  state.savedSupp$ = flyd.flatMap(postSupporter, flyd.map(
+    supporter => ({ ...supporter, tags: state.params$().tags }),
```

Comment:
What happens if `tags` is `null` or `undefined` here? I suspect it will be fine but...

Reply:
I just tested, it worked properly :smiley_cat: